### DISCUSSION TITLE
PR Settings

### DISCUSSION THREAD

**Discussion by BrutuZ** [Animeboynz/Mihon-Backup-Viewer#34] [Ideas]
[https://github.com/Animeboynz/Mihon-Backup-Viewer/settings](https://github.com/Animeboynz/Mihon-Backup-Viewer/settings/#merge-button-settings)
![image](https://github.com/Animeboynz/Mihon-Backup-Viewer/assets/7104931/e7dac6c4-b6fe-4898-882e-9d15c7c4f755)

**Comment by Animeboynz** [comment]
Pls Help
Commit Checks seems to be failing. @BrutuZ @NGB-Was-Taken

**Reply by BrutuZ** [reply]
Disabled checks for now

**Reply by BrutuZ** [reply]
@Animeboynz Can we at least do away with the merge commits filling the main branch history? The PR's pages and history are already kept separately.

**Reply by Animeboynz** [reply]
Wait I'm not too sure what you mean. Is it that want us to use `Squash Commits` instead of `Merge Commits`?
For some reason the default for me when I try to merge is always different, Sometimes its Merge sometimes its Squash. Not too sure why its switching around but I'll remember to switch them back to squash.

**Reply by BrutuZ** [reply]
That's what the [OP](#discussion-6836820) is about, changing the setting for the repo to disable merge commits entirely so it's not used on accident

**Reply by Animeboynz** [reply]
Sweet, I've done that now
